model_security_adjust_sst/security/hr_security.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="group_hr_operator" model="res.groups">
+        <field name="name">Operator</field>
+        <field name="category_id" ref="base.module_category_human_resources"/>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+    </record>
+
+    <record id="hr.group_hr_user" model="res.groups">
+        <field name="implied_ids"
+               eval="[(3, ref('base.group_user')), (4, ref('group_hr_operator'))]"/>
+    </record>
+
+    <record id="operator_hr_employee_perm" model="ir.rule">
+        <field name="name">operator: hr.employee permission</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="groups" eval="[(4, ref('group_hr_operator'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="False"/>
+        <field name="perm_unlink" eval="False"/>
+    </record>
```

Comment:
This record is not needed.  The record in ir.model.access.csv does the job.

Reply:
This part is deleted